f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -230,8 +230,10 @@ def delete(self, loadbalancer, service, **kwargs):
     def _pre_delete(self, service):
         # assign neutron network object in service
         # with route domain first
-        self.driver.network_builder._annotate_service_route_domains(
-            service)
+        # self.driver.network_builder is None in global routed mode
```

Comment:
Make an empty network_builder may be better than asserting `self.driver.network_builder` every time.

Reply:
> Make an empty network_builder may be better than asserting `self.driver.network_builder` every time.

The initialization of network_builder is located in legacy code, I do not hope to change it.